src/main/java/eu/dissco/core/translator/terms/specimen/chronometric/ChronometricAgeDeterminedBy.java

```diff
@@ -0,0 +1,22 @@
+package eu.dissco.core.translator.terms.specimen.chronometric;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import eu.dissco.core.translator.terms.Term;
+import java.util.List;
+
+public class ChronometricAgeDeterminedBy extends Term {
+
+  public static final String TERM = "chrono:chronometricAgeDeterminedBy";
```

Comment:
maybe add the chrono prefix to the Term class? you do that with dwc/ods/dcterms

Reply:
Yeah wasn't sure if we were happy with the prefix part but let's keep it in line with the rest